### PR TITLE
Mention the startup and shutdown sample

### PR DESCRIPTION
**Pull request by danielmarbach**
@Particular/documentation please review

### CODE REVIEW

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ The core will now pass the error queue address to the transport to make it easie
 
 ## IWantToRunWhenBusStartsAndStops execution
 
-Implementations of `IWantToRunWhenBusStartsAndStops` are no longer started and stopped on its dedicated thread. They are executed on the thread starting and disposing the bus. It is the responsibility of the implementing class to execute its operations in parallel if needed (i.ex. for CPU bound work). Failure to do so will prevent the bus from being started and/or disposed.
+Implementations of `IWantToRunWhenBusStartsAndStops` are no longer started and stopped on its dedicated thread. They are executed on the thread starting and disposing the bus. It is the responsibility of the implementing class to execute its operations in parallel if needed (i.ex. for CPU bound work). Failure to do so will prevent the bus from being started and/or disposed. More information about the startup and shutdown sequence can be found in the [Startup and Shutdown sample](/samples/startup-shutdown-sequence/sample.md).
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: not your change but does "on its dedicated thread" sound weird?

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: again not your change `i.e.` not `i.ex.`

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: but the link is fine :) want me to fix the others as part of a merge?

Comment (danielmarbach) [inline nservicebus/upgrades/5to6.md]: Would you say "on a dedicated thread"?

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: `on a dedicated thread` yep thats what i was thinking

**Comment by danielmarbach** [comment]
@SimonCropp changed

**Pull request merged**